instrumentation/opentelemetry-instrumentation-django/src/opentelemetry/instrumentation/django/middleware/otel_middleware.py

```diff
@@ -208,7 +222,24 @@ def process_request(self, request):
         )
 
         attributes = collect_request_attributes(carrier)
-
+        active_requests_count_attrs = {
+            attr_key: attributes[attr_key]
+            for attr_key in _active_requests_count_attrs
+            if attributes.get(attr_key) is not None
+        }
+        duration_attrs = {
+            attr_key: attributes[attr_key]
+            for attr_key in _duration_attrs
+            if attributes.get(attr_key) is not None
+        }
```

Comment:
Use [`_parse_active_request_count_attrs`](https://github.com/TheAnshul756/opentelemetry-python-contrib/blob/25a888f2c26b6975fe628ff6475885a09af9b072/util/opentelemetry-util-http/src/opentelemetry/util/http/__init__.py#L152) and [`_parse_duration_attrs`](https://github.com/TheAnshul756/opentelemetry-python-contrib/blob/25a888f2c26b6975fe628ff6475885a09af9b072/util/opentelemetry-util-http/src/opentelemetry/util/http/__init__.py#L160)

Reply:
Done